frontend/views/containers/payments/MonthOverview.vue

```diff
@@ -54,14 +54,23 @@ export default ({
   },
   computed: {
     ...mapGetters([
+      'currentPaymentPeriod',
+      'dueDateForPeriod',
       'ourGroupProfile',
       'groupSettings',
       'ourPaymentsSummary',
-      'ourPayments'
+      'ourPayments',
+      'periodStampGivenDate'
     ]),
     currency () {
       return currencies[this.groupSettings.mincomeCurrency].displayWithCurrency
     },
+    humanDueDate () {
+      return this.humanDate(this.dueDateForPeriod(this.currentPaymentPeriod))
```

Comment:
You didn't write this code (and I'm not going to hold up the PR review over this), but just want to note that it doesn't make sense to turn `humanDate` into a method of this component - unless you want to use it in the markup (and I'm guessing that way back when that's what was being done and why it was added as a method).

Reply:
Makes sense, I should have reverted `humanDate` to be a plain import